pages/events/index.vue

```diff
@@ -336,6 +359,12 @@ export default class EventsPage extends QiskitPage {
     }
   }
 
+  &__calendar{
+    margin-bottom: $spacing-06;
+    border: 1px solid #8d8d8d;
+    margin-top: $spacing-06;
```

Comment:
I feel like this top margin isn't doing anything... can you please check?

Reply:
it adds a margin on the smaller screens, I can remove it if you like, but it felt right to add it.
Here's with:

<img width="389" alt="Screenshot 2023-02-24 at 11 29 39" src="https://user-images.githubusercontent.com/108661074/221156303-518bfa73-6a4e-4a37-871b-7a3e28384168.png">

and without:

<img width="473" alt="Screenshot 2023-02-24 at 11 29 54" src="https://user-images.githubusercontent.com/108661074/221156403-dcab61f7-92af-4b1c-a1db-749eabe03de6.png">

Reply:
no no, you're right -- it's better w/ the margin for mobile, thanks for pointing that out.

In this case, we can use one of our media query mixins:

```
&__calendar{
    margin-bottom: $spacing-10;
    border: 1px solid $border-color ;

    @include mq($until: medium) {
      margin-top: $spacing-06;
    }
  }
```

this **includes** a media query mixin (**mq**), so that the margin-top style only applies up until the **medium** breakpoint.